lib/websockets/client.py

```diff
@@ -0,0 +1,71 @@
+"""
+Websockets client for micropython
```

Comment:
at the moment the badge store will complain if we have folders within "lib/", it assumes one file=one lib. Could you merge the two?

Reply:
Done in db584da8d4eadf850e783e620865427cd51a999d 👍